code/modules/reagents/chemistry/machinery/chem_master.dm

```diff
@@ -507,7 +507,7 @@
 			var/datum/reagent/master_reagent = reagents.get_master_reagent()
 			if(selected_container == default_container) // Tubes and bottles gain reagent name
 				item_name_default = "[master_reagent.name] [item_name_default]"
-			if(!(initial(selected_container.reagent_flags) & OPENCONTAINER)) // Closed containers get both reagent name and units in the name
+			if(UNLINT(!(initial(selected_container.reagent_flags) & OPENCONTAINER))) // Closed containers get both reagent name and units in the name
```

Comment:
Why?

Reply:
I want(ed) reagent container reagent flags to be protected beacause you should be reading the reagent container's *reagents* reagent flags but this is the only place I couldn't figure out a way around it